packages/client/src/components/Question/Question.tsx

```diff
@@ -84,7 +93,7 @@ const Question: FunctionComponent<QuestionProps> = ({
     <Form
       dataId={questionId}
       dataTestId={QUESTION_FORM}
-      onSubmit={handleSubmit(() => onGoToNext())}
+      onSubmit={handleSubmit(() => onGoToNext && onGoToNext())}
```

Comment:
Can there be a form without a on goToNext?

Reply:
Yup, all questions on the Permit Form topics. That's something to refactor though..